packages/babel-plugin-transform-class-properties/README.md

```diff
@@ -76,11 +76,11 @@ require("babel-core").transform("code", {
 
 ## Options
 
-### `spec`
+### `loose`
 
 `boolean`, defaults to `false`.
 
-Class properties are compiled to use `Object.defineProperty`. Static fields are now defined even if they are not initialized.
+Class properties are compiled use an assignment expression instead of `Object.defineProperty`. Static fields are now not defined if they are not initialized.
```

Comment:
We can put the code example here to be more clear.

so an example of the defineProperty vs. `this.x = a` and also `this.x = undefined`